komfydb/execution/order_by.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __ORDER_BY_H__
+#define __ORDER_BY_H__
+
+#include <vector>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+
+#include "komfydb/execution/op_iterator.h"
+
+namespace komfydb::execution {
+
+class OrderBy : public OpIterator {
+ private:
+  OpIterator* child;
+  TupleDesc td;
+  int order_by_field;
+  bool asc;
```

Comment:
Suggest to add enum ORDER { ASCENDING, DESCENDING } and use that instead of bool.